### PR TITLE
Mw tl rate

### PR DESCRIPTION
**Pull request by mwolfram**
- low-performance launch file: styx_lo.launch (1Hz tl_detector)
- brake deadband is added to brake value that comes from PID controller

### CODE REVIEW

**Review by eljefec — CHANGES_REQUESTED** [review]
I'd expect to add brake_deadband.

**Comment by mwolfram** [comment]
Got confused there, thanks for pointing it out. It's now being added. That would, however, mean that the improvements I'm seeing come from the brake_factor

File: ros/src/twist_controller/launch/dbw.launch [hunk 1]
```diff
@@ -4,6 +4,7 @@
         <param name="vehicle_mass" value="1736.35" />
         <param name="fuel_capacity" value="13.5" />
         <param name="brake_deadband" value=".1" />
+        <param name="brake_factor" value="5.0" />
```
Comment (eljefec) [inline ros/src/twist_controller/launch/dbw.launch]: Justin's graphs on our team channel show our brake values are close to the successful bag brake values. A factor of 5 may drastically change our Carla output. Should this factor be lower for Carla?

**Review by eljefec — CHANGES_REQUESTED** [review]

File: ros/src/twist_controller/launch/dbw.launch (hunk 1 above)
Comment (mwolfram) [reply ros/src/twist_controller/launch/dbw.launch]: I'll set that to 1.0 for Carla, so we can stick to the values found in the bag file. You're right, for the real car it's safer to go easy on the brakes.

**Review by eljefec — APPROVED** [review]

**Pull request merged**